errdefs/http_helpers.go

```diff
@@ -9,6 +9,7 @@ func FromStatusCode(err error, statusCode int) error {
 	if err == nil {
 		return nil
 	}
+
```

Comment:
Oh! Looks like a left-over change from a previous iteration; can you revert this change?